frontend/lib/services/persistency_service.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:async';
+
+import 'package:frontend/models/event.dart';
+import 'package:frontend/models/sailing_club.dart';
+import 'package:frontend/models/boat.dart';
+
+// TODO: Should we have a percistency Interface that is segregated from the authentication interface?
+
+abstract class PersistencyService {
+//  final RegattaStore _store;
+
+//  PersistencyService(this._store);
+
+  Future signIn();
+  void signOut();
+
+  // These private methods are called by implementing classes
+  // to propagate changes in the DB to the frontend.
+/*
+  void newEvent(Event newEvent) {
```

Comment:
The initial reason of defining those in the PercistencyService Class was to decouple the usage of Redux in the frontend from the usage of firebase in the backend.

If this goal is even valid might also need some discussion... :-)